src/main/webapp/stories.html

```diff
@@ -48,6 +48,16 @@ <h4 class="pb-2 mb-1">Ninetailed Personalization Platform</h4>
         </div>
     </div>
 
+    <div class="row m-5">
+        <div class="col">
+            <h4 class="pb-2 mb-1"> Dropsolid - The Digital Experience Company</h4>
+            <p>The Dropsolid experience cloud is not only a cloud platform that allows developers and marketers to work faster and without restrictions of proprietary code, it also delivers an out of box DXP ready enhanced Drupal CMS, Mautic marketing automation platform and Unomi CDP to build powerful user experiences.</p>
+            <p>At Dropsolid CMS, commerce, marketing automation and customer data work seamlessly together.</p>
+             <p>"Open source is in our blood from the start. Apache Unomi was the obvious customer data platform to integrate in our DXP solution." - Nick Veenhof, Dropsolid CTO.</p>
```

Comment:
Maybe talk about potential contributions in the quote?

Reply:
Added paragraph about helping where we can.